cmd/localCI/repo.go

```diff
@@ -94,6 +94,9 @@ type Repo struct {
 
 	// logger of the repository
 	logger *logrus.Entry
+
```

Comment:
Is this patch needed as part of this series. It's not clear to me (I've not read the second patch yet), but there is nothing in the commit message or leaping out from the code at me that it is - just checking it was not a slip of the git ?
and, s/wih/with/ in the commit message